doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -260,8 +281,7 @@ Induction
    In the case of dependent products, the tactic tries to find an instance for
    which the induction principle applies and otherwise fails.
 
-   Most parameters behave as they do in :tacn:`destruct`, with the following
-   differences:
+   Most parameters behave as they do in :tacn:`destruct`:
```

Comment:
Right. So now the question becomes if we want to repeat the information that's already supposed to be available in `destruct`. Up to you.